### DISCUSSION TITLE
Abhängige Suche

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#127] [Q&A]
Das Suchergebnis ändert sich nicht wenn man eine andere Kategorie wählt.

Man wählt eine Kategorie (Gebäudecluster) und klickt auf „Nutzer“ – die Liste wird angezeigt

 
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/cc005586-1eae-403c-82e8-1af86278d8a7)

Wenn man jetzt die Kategorie ändert und wieder auf Nutzer klickt, erscheinen dieselben Einträge wie in der vorigen Suche.

 
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/73cd8849-e387-4146-b5cf-1ffec1e3117f)

Es sollte aber so aussehen:

 
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/9a5d3a5f-00b8-433a-ae8d-c6ed13f89980)

**Comment by gis-eni** [comment]
Du hast recht, da hat sich ein Bug versteckt. Der Tritt auf, wenn man schon die Ergebnisse anzeigt, bevor der Anwender ein Zeichen eingeben hat. Das ist dann mit Build 5.23.5001 gefixt.
